interfaces/builtin/network_control.go

```diff
@@ -233,6 +233,7 @@ capability setuid,
 
 # resolvconf
 /{,usr/}sbin/resolvconf ixr,
+/run/resolvconf/ w,
 /run/resolvconf/{,**} rk,
 /run/resolvconf/** w,
```

Comment:
Sorry for my lack of experience here, but couldn't these 3 be merged? This would reduce the apparmor parser memory usage.

Reply:
Ah just saw that @alexmurray mentioned this, +1 on merging

Reply:
Done now